src/Data/Argonaut/Encode.purs

```diff
@@ -1,10 +1,11 @@
 module Data.Argonaut.Encode
   ( module Data.Argonaut.Encode.Class
   , module Data.Argonaut.Encode.Combinators
+  , module Data.Argonaut.Encode.Implementation
```

Comment:
`Implementation` to me sounds like an internal module that you wouldn't necessarily import into your code. What do you think of renaming the `Implementation` modules to be `Encoders` and `Decoders` instead?

```purs
module Data.Argonaut.Encode
  ( ...
  , module Data.Argonaut.Encode.Encoders
  )
```

```purs
module Data.Argonaut.Decode
  ( ...
  , module Data.Argonaut.Decode.Decoders
  )
```

Reply:
Ok

Reply:
done